.env

```diff
@@ -1,2 +1,7 @@
 # .env
-DISCORD_TOKEN = ###TOKEN HERE###
\ No newline at end of file
+
+DISCORD_TOKEN = ###TOKEN HERE###
+
+BACKGROUND_LOOP_TIME = 60  # seconds of running loop for party cleanup
```

Comment:
Re: https://github.com/danny-tom/charon/pull/13#issuecomment-684035192

Unrelated to content, but the suggestion feature in Github is really handy for changes like this:

```suggestion
# seconds of running loop for party cleanup
BACKGROUND_LOOP_TIME = 60
```

Reply:
cool, I just figured out how to do it, SUPER USEFUL. Thanks a ton, Caspar!

Reply:
Danny, this review is because the comment in .env is being included in the environment variable BACKGROUND_LOOP_TIME. move the comment to the line before.